### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by tyfoo**
I've made a couple of minor changes:
- Removed the require of a less file as it caused issues.
- Added an ability/prop to pass an on focus event handler to the telephone input field.
- Added an event listener to trigger setting focus on telephone input field (trigger by creating an firing an event from the parent node)

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Removing the less file is *not* a minor change :) You can do something instead like surrounding the require with a `if (typeof document !== "undefined")`.

You should add the `onFocus` handler to the actual element instead of creating a global one.

**Comment by tydicefm** [comment]
Thanks for the quick feedback.

The reason I commented it out instead of wrapping it with an if is because when I did that, it throws an exception on the client: Error: Cannot find module "../less/default.less".

Also, can you please elaborate on the onFocus comment? There was an event attached to the element already, I just added a way to pass through an external handle function:
```javascript
handleInputFocus(e) {
        // if the input is blank, insert dial code of the selected country
        if(this.refs.numberInput.getDOMNode().value === '+') {
            this.setState({formattedNumber: '+' + this.state.selectedCountry.dialCode});
        }
        if(typeof this.props.onFocus === 'function') {
            this.props.onFocus(e);
        }
    },
```
The 2nd change I've made was to allow setting focus on the nested input element from external source (i.e. a [button click from a different react component]( https://facebook.github.io/react/tips/communicate-between-components.html)).

**Comment by KyleAMathews** [comment]
I'm assuming you're getting an error with less because you don't have less compilation setup for your project?

We should probably compile the less to raw css to make it simpler for people to import as more people have their webpack/browserify projects setup to work with css than less.

File: lib/index.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ var ReactTelephoneInput = React.createClass({
     },
     componentDidMount: function componentDidMount() {
         document.addEventListener('keydown', this.handleKeydown);
+        document.addEventListener('focus:' + this.props.name, this.onFocusEventHandler);
```
Comment (KyleAMathews) [inline lib/index.js]: What are you trying to accomplish here?

Comment (tydicefm) [inline lib/index.js]: The idea here is to allow external components/events set focus on the nested input field. For example, if you have an 'hidden form' with the react-telephone-input component that is displayed on a button click, you can display the form and set the focus on the nested input field directly.

Comment (KyleAMathews) [inline lib/index.js]: This should happen in a wrapper component. Render the button first and listen for a click on the button at which point you hide the button, show the telephone field and set its focus.

**Comment by mukeshsoni** [comment]
Closing this since there are other ways around it.

**Pull request closed without merging**